### ISSUE TITLE
Tagged releases and changelog

### ISSUE THREAD

**Issue by baimafeima** [chronitis/curseradio#5]
Could you tag your releases and make the changelog directly visible in your GitHub releases? This would increase transparency for potential users and distribution maintainers. Something like https://skywinder.github.io/github-changelog-generator/ could be used to generate the changelog based on merged PRs and closed issues since the last release tag.

**Comment by chronitis** [comment]
Yes, this is a fair point. As you will notice, there isn't a changelog or releases to speak of for quite a while, but I'm trying to do some updates at the moment.

**Comment by jubalh** [comment]
Would be indeed nice to have a git tag that you could turn into a release.
I intended to package this too, but like this it's not possible at the moment.
